packages/flame/lib/src/components/inherited_component.dart

```diff
@@ -0,0 +1,3 @@
+import 'package:flame/components.dart';
+
+abstract class InheritedComponent extends Component {}
```

Comment:
Shall this be just a typed mixing? Any suggestions on improving this?

```suggestion
mixin InheritedComponent {}
```